test/SlowTests/SlowTests.csproj

```diff
@@ -32,6 +32,8 @@
     <None Remove="Data\RavenDB_13759\Orders_ByOrderBy.zip" />
     <None Remove="Data\RavenDB_13940\storage-with-reused-journal-and-synced-data-2.zip" />
     <None Remove="Data\RavenDB_13940\storage-with-reused-journal-and-synced-data.zip" />
+    <None Remove="Data\RavenDB_15223\counters.4.2.103.ravendb-snapshot" />
```

Comment:
This file has 40MB. If you need to create this anyway, then we can use Interversion tests to generate this on the fly.

Reply:
I accidentally used a snapshot that was creadted with no compression. 
changed to optimal compression and now this file is 300KB